src/main/java/ru/sbt/mipt/oop/ManageAllEvents.java

```diff
@@ -0,0 +1,13 @@
+package ru.sbt.mipt.oop;
+
+import static ru.sbt.mipt.oop.GetNextSensorEvent.getNextSensorEvent;
+import static ru.sbt.mipt.oop.ManageEvent.manageEvent;
+
+public class ManageAllEvents {
+    static void manageAllEvents(SmartHome smartHome, SensorEvent event) {
```

Comment:
https://github.com/iagoshkov/smart-home-2020/blob/master/STUDENT_GUIDE.md#%D0%BD%D0%B5-%D0%B8%D1%81%D0%BF%D0%BE%D0%BB%D1%8C%D0%B7%D1%83%D0%B9%D1%82%D0%B5-%D1%81%D1%82%D0%B0%D1%82%D0%B8%D1%87%D0%B5%D1%81%D0%BA%D0%B8%D0%B5-%D0%BC%D0%B5%D1%82%D0%BE%D0%B4%D1%8B-%D0%B2-%D0%B1%D0%B8%D0%B7%D0%BD%D0%B5%D1%81-%D0%BB%D0%BE%D0%B3%D0%B8%D0%BA%D0%B5